.github/workflows/cd.yml

```diff
@@ -0,0 +1,50 @@
+name: cd
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  release-next:
+    name: release-next
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: "0"
+      - run: git fetch origin +refs/tags/*:refs/tags/*
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: 14.x
+          registry-url: https://registry.npmjs.org/
+      - name: Install dependencies
+        run: "bash setup_${{ runner.os }}.sh"
+      - run: npm ci
+      - run: npm run build
+      - name: Run unit/integration tests
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: "bash run_${{ runner.os }}.sh"
+      - name: Send results to SonarCloud
+        if: runner.os == 'Linux'
+        uses: SonarSource/sonarcloud-github-action@v1.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
```suggestion
```
I think it is not required to send coverage information on cd builds. At the end, cd just makes sure that a new next release is created from the merged pr. There should not be any code change on develop.